src/test/kotlin/no/skatteetaten/aurora/clerk/controller/ApplicationDeploymentDetailsControllerTest.kt

```diff
@@ -30,65 +28,53 @@ class ApplicationDeploymentDetailsControllerTest : AbstractSecurityControllerTes
     @MockBean
     lateinit var podService: PodService
 
-    val name = "luke"
-    val started = Instant.now().toString()
-    val namespace = "jedi-test"
+    private val name = "luke"
+    private val started = Instant.now().toString()
+    private val namespace = "jedi-test"
 
-    val luke = PodItem("$name-1", name, started, "Running")
+    private val luke = PodItem("$name-1", name, started, "Running")
 
-    val yoda = PodItem("yoda-1", "yoda", started, "Running")
+    private val yoda = PodItem("yoda-1", "yoda", started, "Running")
 
     @Test
     @WithUserDetails
     fun `should get error if incorrect namespace in token`() {
-
-        mockMvc.perform(
-            get("/api/pods/{namespace}", "sith")
-                .header("Authorization", "Bearer <token>")
-        )
-            .andExpect(status().isUnauthorized)
-            .andExpect(jsonPath("$.success", `is`(false)))
-            .andExpect(jsonPath("$.message", `is`("Only an application in the same namespace can use clerk.")))
-            .andDo(document("error-pods"))
+        mockMvc.get(
```

Comment:
named parameters tror jeg blir fint her.

Reply:
Ser ikke ut til at det funker pga vararg

Reply:
Jeg har sett mange bruke Array<String> istedet for vararg, det er sikkert grunnen.